client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -262,107 +279,92 @@ const InvestigatedPersonInfo = (props: Props) => {
             </div>
             <div className={classes.informationBar}>
                 <div className={classes.additionalInfo}>
+                    <div className={classes.line}>
                     {
                         age !== null &&
                         <>
-                            <InfoItemWithIcon testId='age' name='גיל' value={+age < 1 ? 'פחות משנה' : age} icon={CakeOutlined} />
+                            <PatientInfoItem testId='age' name='גיל' value={+age < 1 ? 'פחות משנה' : age} />
                             {
                                 +age <= maxComplexityAge && <ComplexityIcon tooltipText='המאומת מתחת לגיל 14' />
                             }
-                            <Divider />
                         </>
                     }
-                    <InfoItemWithIcon testId='birthdate' name='תאריך לידה' value={
-                        birthDate ? format(new Date(birthDate), displayDateFormat) : 'אין תאריך'
-                    }
-                        icon={CalendarToday}
-                    />
                     {
                         isMandatoryInfoMissing && <ComplexityIcon tooltipText='אימות מרשם נכשל' />
                     }
-                    <Divider />
-                    <InfoItemWithIcon testId='examinationDate' name='תאריך תחילת מחלה' value=
+                    <PatientInfoItem testId='examinationDate' name='תחילת מחלה' value=
                         {
                             format(validationDate, displayDateFormat)
                         }
-                        icon={EventOutlined}
-                    />
-                    <Divider />
-                    <InfoItemWithIcon testId='gender' name='מין' value={gender}
-                        icon={Help}
-                    />
-                    <Divider />
-                    <InfoItemWithIcon testId='idType' name='סוג תעודה מזהה' value={identityType}
-                        icon={Help}
-                    />
-                    <Divider />
-                    <InfoItemWithIcon testId='idNumber' name='מספר תעודה מזהה' value={identityNumber}
-                        icon={Help}
-                    />
-                    <Divider />
-                    <InfoItemWithIcon testId='isDeceased' name='האם נפטר' value={indication((isDeceased || investigationStatus.subStatus === InvestigationComplexityByStatus.IS_DECEASED))}
-                        icon={Help}
                     />
+                    <PatientInfoItem testId='gender' name='מין' value={gender==='נקבה'?'נ':'ז'} />
```

Comment:
isn't the value returns always female?

Reply:
I think it just looks like it because of RTL

Reply:
👍